tsrc/gitlab.py

```diff
@@ -115,6 +116,12 @@ def create_merge_request(self, project_id, source_branch, *, title,
         ui.info("done", ui.check)
         return result
 
+    def get_merge_request_url(self, merge_request):
```

Comment:
does not seem to be used here. Maybe should be in a separate commit ?

Reply:
removed